src/main/java/net/masterthought/cucumber/generators/EscapeHtmlReference.java

```diff
@@ -2,16 +2,27 @@
 
 import org.apache.commons.lang.StringEscapeUtils;
 import org.apache.velocity.app.event.ReferenceInsertionEventHandler;
+import org.owasp.html.HtmlPolicyBuilder;
+import org.owasp.html.PolicyFactory;
 
 /**
  * Escapes all html and xml that was provided in a reference before inserting it into a template.
+ *
+ * References that start with $_sanitize_ will be sanitized to allow urls.
  */
 final class EscapeHtmlReference implements ReferenceInsertionEventHandler {
 
+    private static final PolicyFactory LINKS = new HtmlPolicyBuilder()
+            .allowStandardUrlProtocols().allowElements("a").allowAttributes("href")
+            .onElements("a").requireRelNofollowOnLinks().requireRelsOnLinks("noopener", "noreferrer")
+            .toFactory();
+
     @Override
     public Object referenceInsert(String reference, Object value) {
         if (value == null) {
             return null;
+        } else if(reference.startsWith("$_sanitize_")) {
```

Comment:
@mpkorstanje this two lines are not covered by tests. Would you like to add test for this?